src/components/ScreenWrapper.tsx

```diff
@@ -190,7 +190,17 @@ function ScreenWrapper(
 
     return (
         <SafeAreaConsumer>
-            {({insets, paddingTop, paddingBottom, safeAreaPaddingBottomStyle}) => {
+            {({
+                insets = {
```

Comment:
Why was it updated?

Reply:
@VickyStash without this change we're required to set default values for insets or do things like `insets?.top ?? 0` in every wrapped component. I think this way is easier 😄